packages/ui/src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { PlusIcon } from '@heroicons/react/24/solid';
+import type { Meta, StoryFn } from '@storybook/react';
+
+import { ButtonSize, ButtonVariants } from './types';
+
+import { Button, type ButtonProps } from '.';
+
+const Icon = PlusIcon;
+
+export default {
+    title: 'Buttons/Button',
+    component: Button,
+} as Meta<typeof Button>;
+
+export const AllVariations: StoryFn<typeof Button> = (args) => {
```

Comment:
Do we really need this story? You can still play around with props on any story. Also, this story doesn't even fit on a screen, so it's hard to overview or snapshot test anyway.

I'd rather make a couple extra stories for disabled/loading state. You can also use `autodocs` to generate an overview page with all stories.